wof/examples/production_configs/wofpy.conf

```diff
@@ -1,5 +1,5 @@
 # description of the script purpose
-description "uWSGI server instance configured to serve timeseries"
+description "uWSGI server instance configured to serve WOFpy"
```

Comment:
Can you point out where that generalization was done? Your in-line comment only refers to a description, and I don't see anything obvious in the rest of the file that points to that generalization.